metricproviders/job/job.go

```diff
@@ -73,8 +73,20 @@ func (p *JobProvider) GetMetadata(metric v1alpha1.Metric) map[string]string {
 // fit into a 63 character label, since the k8s job controller incorporates the job name into the
 // pod spec labels.
 func newJobName(run *v1alpha1.AnalysisRun, metric v1alpha1.Metric) string {
+	const Kubernetes_DNS_Limit = 63
```

Comment:
Can we move this to utils/defaults/defaults.go

Reply:
Done